docs/csharp/index.yml

```
@@ -0,0 +1,103 @@
+### YamlMime:Landing
+
+title: "C# programming language documentation"
+summary: "Learn how to write all kinds of applications using the C# programming language on the .NET platform."
+
+metadata:
+  title: "C# programming language documentation - get started, tutorials, reference."
+  description: "Learn C# programming - for beginning developers, developers new to C#, and experienced C# / .NET developers"
+  ms.topic: landing-page # Required
+  author: billwagner
+  ms.author: wiwagn
+  ms.date: 11/19/2019
+
+# linkListType: | concept | get-started | how-to-guide | overview  | reference | tutorial | video | whats-new
+
+landingContent:
+  - title: "Learn to program in C#"
+    linkLists:
+      - linkListType: get-started
+        links:
+          - text: Online, interactive Hello world
+            url: tutorials/intro-to-csharp/hello-world.yml
+          - text: "Math and numbers in C#"
+            url: tutorials/intro-to-csharp/numbers-in-csharp.yml
+          - text: Introductory tutorials
+            url: tutorials/intro-to-csharp/index.md
+      - linkListType: video
+        links:
+          - text: "What is C#?"
+            url: https://channel9.msdn.com/Series/CSharp-101/What-is-C/player
+          - text: "C# 101 video series"
+            url: https://aka.ms/dotnet3-csharp
+
+  - title: "C# language fundamentals"
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: "A tour of C#"
+            url: tour-of-csharp/index.md
+          - text: "Inside a C# program"
+            url: programming-guide/inside-a-program/index.md
+          - text: Main() and command-line Arguments
+            url: programming-guide/main-and-command-args/index.md
+      - linkListType: concept
+        links:
+          - text: "The C# and .NET type system"
+            url: programming-guide/types/index.md
+          - text: Language Integrated Query (LINQ)
+            url: linq/index.md
+          - text: async and await
+            url: programming-guide/concepts/async/index.md
+      - linkListType: tutorial
+        links:
+          - text: Object oriented inheritance
+            url: tutorials/inheritance.md
+          - text: Use LINQ to query data
+            url: tutorials/working-with-linq.md
+          - text: Format strings using interpolation
```

Not sure if it was by design, only thought that "string interpolation" can be more searchable term
```suggestion
          - text: Format strings using string interpolation
```

---

I agree with your thinking here, but I didn't want "string" to appear twice. I made a different update.